src/services/sync/strategies/SyncMessageBuilder.js

```diff
@@ -0,0 +1,77 @@
+const LDAP_SYNC_ACTIONS = {
+	SYNC_USER: 'syncUser',
+	SYNC_SCHOOL: 'syncSchool',
+	SYNC_CLASSES: 'syncClasses',
+};
+
+class SyncMessageBuilder {
```

Comment:
Must we validate if all 3 called methodes has all needed keys, or is this a part that must handled by the consumer?
I do not know the best pratcis way in the producer <-> consumer stack.